test/lib/mayaUsd/render/mayaToHydra/testMayaDisplayModes.py

```diff
@@ -48,42 +48,51 @@ def test_MayaDisplayModes(self):
                 "testMayaDisplayModes",
                 "testMayaDisplayModes.ma")
         cmds.refresh()
-
         panel = mayaUtils.activeModelPanel()
         
+        
         cmds.modelEditor(panel, edit=True, wireframeOnShaded=False)
 
         #Use Default Material
         cmds.modelEditor(panel, edit=True, useDefaultMaterial=True)
         cmds.refresh()
         self.assertSnapshotClose("defaultMaterial" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        cmds.refresh()
+
+        #xray mode
+        cmds.modelEditor(panel, edit=True, useDefaultMaterial=False)
+        cmds.modelEditor(panel, edit=True, xray=True)
+        cmds.refresh()
+        self.assertSnapshotClose("xray" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)        
+        cmds.modelEditor(panel, edit=True, xray=False)
 
+        #Use Default Material
+        cmds.modelEditor(panel, edit=True, useDefaultMaterial=True)
+        cmds.refresh()
+        self.assertSnapshotClose("defaultMaterial" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
+        
         #WireframeOnShaded mode
         cmds.modelEditor(panel, edit=True, wireframeOnShaded=True)
         cmds.refresh()
         self.assertSnapshotClose("wireframeOnShaded" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
         cmds.modelEditor(panel, edit=True, wireframeOnShaded=False)
-
+        
         #Shadows mode
         cmds.modelEditor(panel, edit=True, shadows=True)
         cmds.refresh()
         self.assertSnapshotClose("shadows" + ".png", self.imageDiffFailThreshold, self.imageDiffFailPercent)
         cmds.modelEditor(panel, edit=True, shadows=False)
-
+                
         #All Lights mode
         cmds.modelEditor(panel, edit=True, displayLights="all")
```

Comment:
You should reset the value of displayLights to what it was before that call at line 88.